appveyor.yml

```diff
@@ -1,12 +1,21 @@
 platform:
-  - x86
   - x64
+  - x86
+
 configuration:
   #- Debug
   - Release
 
+environment:
+  CMAKE: cmake.exe
```

Comment:
We depend on a patch in cmake. We need recent cmake (later than 01 Aug 2016 nightly build). Cpt installs it.